File: RXCareServer/RXCareServer/Models/Comment.cs
namespace RXCareServer.Models
{
    public class Comment
    {
        public int? Id { get; set; }
        public int? PatientId { get; set; }
        public int? MedicineId { get; set; }
        public string? PComment { get; set; }
        public DateTime? PCommentDate { get; set; }
        public string? DComment { get; set; }
        public DateTime? DCommentDate { get; set; }
    }

    public class CommentInfo
    {
        public int? Id { get; set; }
        public int? PatientId { get; set; }
        public int? MedicineId { get; set; }
        public string?   PComment { get; set; }
        public DateTime? PCommentDate { get; set; }
        public string? DComment { get; set; }
        public DateTime? DCommentDate { get; set; }
        public Medicine? Medicine { get; set; }
    }
}

File: RXCareServer/rxcareclient/src/components/comments/Comment.js
import React, { useEffect, useState } from "react";
import { useNavigate } from "react-router-dom";
import "../profile/ProfilePatient.css";
import Accordion from "react-bootstrap/Accordion";

export const Comment = ({
    patient_Id,
    medicineId,
    pComment,
    pCommentDate,
    dComment,
    dCommentDate,
    medicineName,
}) => {
    return (
        <>
            <Accordion.Item eventKey="1">
                <Accordion.Header>COMMENTS</Accordion.Header>
                <Accordion.Body>
                    {<h5>medicineName:{medicineName}</h5>}
                    <p>P:{pComment}</p>
                    <p>D:{dComment}</p>
                </Accordion.Body>
            </Accordion.Item>
        </>
    );
};
